Dockerfile

```diff
@@ -7,14 +7,21 @@ LABEL org.opencontainers.image.source https://github.com/serengil/deepface
 RUN mkdir /app
 RUN mkdir /app/deepface
 
+
+
 # -----------------------------------
 # switch to application directory
 WORKDIR /app
 
 # -----------------------------------
 # update image os
-RUN apt-get update
-RUN apt-get install ffmpeg libsm6 libxext6 -y
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    ffmpeg \
+    libsm6 \
+    libxext6 \
+    libhdf5-dev \
```

Comment:
what is the purpose of installation of libhdf5-dev?

Reply:
@serengil I install libhdf5-dev to prevent h5py package failed to build sir.